static/js/modules/tables.js

```diff
@@ -222,7 +222,7 @@ function compareQuery(first, second, keys) {
   return !different;
 }
 
-function pushQuery(params) {
+function pushQuery(params, push) {
```

Comment:
I'm less persuaded by the no-booleans argument for languages that support named function parameters:

```python
my_function(flag_1=True, flag_2=False, ...)
```

And ES6 will sort of support this syntax with parameter destructuring. But, since we're not using ES6 here (yet!), agreed--I'll update the patch.